search/app.js

```diff
@@ -74,7 +76,9 @@ const getDocs = async (event, parsedPath) => {
 
 const getRoot = async (event, parsedPath) => {
   console.log(`getRoot ${JSON.stringify(parsedPath, null, 2)}`);
-  const accept = _.get(event, 'headers.Accept', 'application/json');
+  // TODO: remove lodash
+  const accept = Object.values(event.headers.Accept,)
```

Comment:
We are going to have to address this with a new get method we implement. I would like to create this with tests, so this will have to wait until we can pair.

Reply:
Ok, sounds good.